packages/api/v1/api-documentation.tsx

```diff
@@ -1,11 +1,30 @@
 'use client';
 
+import { useEffect } from 'react';
+
+import { useTheme } from 'next-themes';
 import SwaggerUI from 'swagger-ui-react';
 import 'swagger-ui-react/swagger-ui.css';
 
 import { OpenAPIV1 } from '@documenso/api/v1/openapi';
 
 export const OpenApiDocsPage = () => {
+  const { theme } = useTheme();
```

Comment:
I think we should use `resolvedTheme` in this case.

> resolvedTheme: If enableSystem is true and the active theme is "system", this returns whether the system preference resolved to "dark" or "light". Otherwise, identical to theme
*[source](https://github.com/pacocoursey/next-themes#:~:text=resolvedTheme%3A%20If%20enableSystem%20is%20true%20and%20the%20active%20theme%20is%20%22system%22%2C%20this%20returns%20whether%20the%20system%20preference%20resolved%20to%20%22dark%22%20or%20%22light%22.%20Otherwise%2C%20identical%20to%20theme)*

Or am I wrong? What do you think?

Reply:
Makes sense, I have sent a commit below to update it :)